### ISSUE TITLE
About backbone

### ISSUE THREAD

**Issue by zhengjiawen** [SakuraRiven/EAST#1]
你好，我想请教下，你有试过使用ResNet50或者ResNeXt等作为backbone吗，比起VGG16这些网络的效果应该会更好吧？
另外，我想请问下，你复现的EAST是否用到一些trick，比起 @argman 的https://github.com/argman/EAST performance还提升了一些。
非常感谢！

**Comment by SakuraRiven** [comment]
1. Sry I didn't test on other backbones. The PyTorch has pretrained models which can be directly used so you can try  : )

2. I didn't perform the ablation experiments and here just list part of details (maybe not tricks) in this implementation:
(1) The geo loss is averaged on valid gt region instead of entire batch in @argman, so the weight of geo loss and cls loss is equal in this repo;
(2) The ignored map (for ### text) is used;
(3) The data aug includes rotate、adjust ratio, random crop and color jitter. During the crop, I repeated many times to avoid cutting text regions;
(4) Some detect params are optimized according to test performance... ZZZ

More details could be found in codes.

**Comment by zhengjiawen** [comment]
> 1. Sry I didn't test on other backbones. The PyTorch has pretrained models which can be directly used so you can try  : )
> 2. I didn't perform the ablation experiments and here just list part of details (maybe not tricks) in this implementation:
>    (1) The geo loss is averaged on valid gt region instead of entire batch in @argman, so the weight of geo loss and cls loss is equal in this repo;
>    (2) The ignored map (for ### text) is used;
>    (3) The data aug includes rotate、adjust ratio, random crop and color jitter. During the crop, I repeated many times to avoid cutting text regions;
>    (4) Some detect params are optimized according to test performance... ZZZ
> 
> More details could be found in codes.

Thanks!

**Comment by SakuraRiven** [comment]
Welcome to share the performances when other backbones are used.

**Comment by zhengjiawen** [comment]
Hi, @SakuraRiven , this is my result in resnet50 and resnet101:

Method | Precision(%) | Recal(%)l | Hmean(%)
-- | -- | -- | --
EAST   VGG16 Origin paper | 80.46 | 72.75 | 76.41
EAST   VGG16 reimplement | 80.02 | 80.45 | 80.24
EAST   ResNet50 | 80.50 | 79.92 | 80.21
EAST   ResNet101 | 81.15 | 80.64 | 80.89

I run your code and only get 80.24 H-mean, and I train with resnet50 and 101, the performances are not as good as I had expected. Maybe there's something wrong with my parameters. Thanks for your code, and I will refine my model.

**Comment by SakuraRiven** [comment]
@zhengjiawen Thanks for your work! Actually, the best model is from ~400 epoch in my experiment and lr=1e-5 doesn't help. So in the new code, the last stage (600-900 epoch) is removed. You can search the best model in [300-500] epoch.

**Comment by WeihongM** [comment]
> Hi, @SakuraRiven , this is my result in resnet50 and resnet101:
> 
> Method	Precision(%)	Recal(%)l	Hmean(%)
> EAST VGG16 Origin paper	80.46	72.75	76.41
> EAST VGG16 reimplement	80.02	80.45	80.24
> EAST ResNet50	80.50	79.92	80.21
> EAST ResNet101	81.15	80.64	80.89
> I run your code and only get 80.24 H-mean, and I train with resnet50 and 101, the performances are not as good as I had expected. Maybe there's something wrong with my parameters. Thanks for your code, and I will refine my model.

When use Resnet as backbone network, how you change the channel setting, which is different from VGG network?

**Comment by WeihongM** [comment]
When use Resnet as backbone network, how you change the channel setting, which is different from VGG network?
The Vgg network backbone have different channels output compared with resnet.
@zhengjiawen  @SakuraRiven

**Comment by 123cjjjj** [comment]
> When use Resnet as backbone network, how you change the channel setting, which is different from VGG network? The Vgg network backbone have different channels output compared with resnet. @zhengjiawen @SakuraRiven

I have the same question ,do you have solved it?
